ds-controller-board/driver/driver.go

```diff
@@ -131,7 +133,7 @@ func (drv *ControllerBoardDriver) HandleWriteCommands(deviceName string, protoco
 				return err
 			}
 		default:
-			return fmt.Errorf("unknown Command Type: '%d'", cmdType)
+			return fmt.Errorf("unknown Command Type: '%v'", cmdType)
```

Comment:
Good change. We don't know what this type will be so allowing go to do the casting is much better